CHANGELOG.md

```diff
@@ -2,9 +2,6 @@
 
 ## Next Version
 
-#### Fixed
-- Fixed no such mocule `DOT` error [#1065](https://github.com/yonaskolb/XcodeGen/pull/1065) @yanamura
```

Comment:
If we pin the version below I believe this should still be fixed, so you an add it back with this PR, but fix `mocule` -> `module`